omegaconf/omegaconf.py

```diff
@@ -937,7 +937,7 @@ def _node_wrap(
             element_type=element_type,
         )
     elif type_ == Any or type_ is None:
-        node = AnyNode(value=value, key=key, parent=parent, is_optional=is_optional)
+        node = AnyNode(value=value, key=key, parent=parent)
```

Comment:
I actually did that before submitting the PR (though the correct assert is `assert is_optional`), the only failures are in a few tests that explicitly call `_node_wrap()` with `optional` set to False. It would be a bit annoying to fix because they are part of parameterized tests, so I let it go.

Reply:
roger.